.jenkins

```diff
@@ -27,17 +27,18 @@ pipeline {
         }
         stage('Build') {
             parallel {
-                stage('OPENACC-NVHPC-CUDA-11.6') {
+                stage('OPENACC-NVHPC-CUDA-12.2') {
                     agent {
                         dockerfile {
                             filename 'Dockerfile.nvhpc'
                             dir 'scripts/docker'
+                            additionalBuildArgs '--build-arg BASE=nvcr.io/nvidia/nvhpc:23.7-devel-cuda12.2-ubuntu20.04'
```

Comment:
Please update https://github.com/kokkos/kokkos/blob/1a3ea28f6e97b4c9dd2c8ceed53ad58ed5f94dfe/scripts/docker/Dockerfile.nvhpc#L1

Reply:
It is fine to specify the build argument here but the rule of thumb is that images built without build arguments should also be usable.